src/rpc/blockchain.cpp

```diff
@@ -1455,9 +1455,9 @@ UniValue getblockchaininfo(const JSONRPCRequest& request)
     softforks.push_back(SoftForkDesc("bip65", 4, tip, consensusParams));
     BIP9SoftForkDescPushBack(bip9_softforks, "csv", consensusParams, Consensus::DEPLOYMENT_CSV);
     BIP9SoftForkDescPushBack(bip9_softforks, "dip0001", consensusParams, Consensus::DEPLOYMENT_DIP0001);
+    BIP9SoftForkDescPushBack(bip9_softforks, "bip147", consensusParams, Consensus::DEPLOYMENT_BIP147);
```

Comment:
If the intent of this change is to put them in activation order, maybe there should also be a comment somewhere above saying that? Otherwise I can see how it would be easy to get moved again if someone is putting in e.g. alphabetical order.